**Exercício** teste z

**A altura média da turma On35 é diferente da altura média nacional da população feminina brasileira?**

**Hipótese Nula (H0):**

Não há diferença entre a média da amostra e a média da população (161 cm). A média da altura das alunas dessa turma é 161 cm.

**Hipótese Alternativa (H1):**

Existe uma diferença entre a média da amostra e a média da população. A média da altura das alunas dessa turma não é 161 cm.

**Dados da amostra**
[157, 160, 157, 163, 170, 167, 174, 160, 169, 158, 161, 168, 162, 167, 167, 165, 170, 170]

**Teste**

In [1]:
import numpy as np
from statsmodels.stats.weightstats import ztest

altura_turma_on35 = np.array([157, 160, 157, 163, 170, 167, 174, 160, 169, 158, 161, 168, 162, 167, 167, 165, 170, 170])
media_nacional = 161

z_statistic, p_value = ztest(altura_turma_on35, value=media_nacional)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}, p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: 3.0776, p-value: 0.0021


**Interpretação dos resultados**


Estatística z: 3.0776

Valor p (p-value): 0.0021


Para o nível de Significânci 0.05 (ou 5%) 
O valor p 0.0021 é menor que 0.05. Então, rejeitamos a hipótese nula.


Sendo assim, conclui-se que a altura média da amostra de alturas dessa turma é significativamente diferente de 161 cm. Então, existem evidências estatisticamente significativas para afirmar que a altura média da turma difere da altura média nacional brasileira feminina de 161 cm.
